src/root/dsystem.h

```diff
@@ -0,0 +1,65 @@
+
+/* Compiler implementation of the D programming language
+ * Copyright (C) 1999-2018 by The D Language Foundation, All Rights Reserved
+ * written by Walter Bright
+ * http://www.digitalmars.com
+ * Distributed under the Boost Software License, Version 1.0.
+ * http://www.boost.org/LICENSE_1_0.txt
+ * https://github.com/dlang/dmd/blob/master/src/dmd/root/dsystem.h
+ */
+
+#pragma once
+
+// Get common system includes from the host.
+
+#define POSIX (__linux__ || __GLIBC__ || __gnu_hurd__ || __APPLE__ || __FreeBSD__ || __OpenBSD__ || __sun)
```

Comment:
If this is supposed to be up to date with the master branch then there's DragonFly BSD as well.

Reply:
I don't see DragonflyBSD in master.

Reply:
Seems like you removed it recently: https://github.com/dlang/dmd/commit/d1bfdbca.

Reply:
Ah, it's in other sources.  Probably worth putting it in a separate after this if still needed.

Reply:
Oh, heh, copy-pasto.